src/main/kotlin/persistence/injection/PersistenceModule.kt

```diff
@@ -0,0 +1,22 @@
+package persistence.injection
+
+import dagger.Module
+import dagger.Provides
+import data.persistence.AppDatabase
+import data.persistence.AppPreferences
+import persistence.AppDatabaseImpl
+import persistence.AppPreferencesImpl
+import persistence.DirectoryProvider
+import persistence.IDirectoryProvider
+
+@Module
+class PersistenceModule {
+    @Provides
+    fun providesAppDatabase() : AppDatabase = AppDatabaseImpl
+
+    @Provides
+    fun providesAppPreferences() : AppPreferences = AppPreferencesImpl
+
+    @Provides
+    fun providesDirectoryProvider() : IDirectoryProvider = DirectoryProvider("OralTranslationRecorder")
```

Comment:
Let's stick with TranslationRecorder for now, if we need the appname to be provided here